spec/Powertrain/EngineCoolant.vspec

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2016 Contributors to COVESA
```

Comment:
2016 -> 2024?

Reply:
In one way 2016 can make sense, as some of the signals actually have been copied from another file and maybe they were added in 2016, I have not checked.  So even if the file is created 2024, the content is older